src/services/services.ts

```diff
@@ -2912,6 +2919,63 @@ module ts {
 
                 return filteredMembers;
             }
+
+            function isExternalModuleName(token: Node): boolean {
+                if (token && token.parent && token.kind === SyntaxKind.StringLiteral && isInStringOrRegularExpressionOrTemplateLiteral(token)) {

```

Comment:
But it *is* a literal. What does it mean for a literal to be *in* another literal? (Besides a literal nested in a template substitution expression, which is not interesting here).